src/Symfony/Component/Console/Helper/QuestionHelper.php

```diff
@@ -46,13 +46,23 @@ public function ask(InputInterface $input, OutputInterface $output, Question $qu
         }
 
         if (!$input->isInteractive()) {
-            if ($question instanceof ChoiceQuestion) {
+            $default = $question->getDefault();
+
+            if (null !== $default && $question instanceof ChoiceQuestion) {
                 $choices = $question->getChoices();
 
-                return $choices[$question->getDefault()];
+                if (!$question->isMultiselect()) {
+                    return isset($choices[$default]) ? $choices[$default] : $default;
+                }
+
+                $default = explode(',', $default);
+                foreach ($default as $k => $v) {
+                    $v = trim($v);
+                    $default[$k] = isset($choices[$v]) ? $choices[$v] : $v;
+                }
```

Comment:
Borrowed from https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Console/Helper/SymfonyQuestionHelper.php#L47 so that it's consistent everywhere